fedora-vbx.md

```diff
@@ -0,0 +1,74 @@
+### Install Vagrant using VirtualBox on Fedora 24
+
+Tested for Fedora 24
+
+Run the following commands on fresh installed Fedora first:
+```shell
+# Upgrade fedora
+sudo dnf -y update          # upgrade to the latest kernel etc.
+
+# Install vim editor
+sudo dnf -y install vim
+
+# Reboot machine
+sudo reboot
+```
+
+Have you run `dnf update` lately, then start here:
+```shell
+# Generate SSH key to use when accessing machines
+ssh-keygen -t rsa -b 4096 -N "" -C "root@example.com" -f ~/.ssh/rhce # empty password
+
+# Make loading SSH key to ssh agent automatic
+eval $(ssh-agent); echo $'eval $(ssh-agent)' >> ~/.bash_profile
+ssh-add ~/.ssh/rhce; echo $'ssh-add ~/.ssh/rhce' >> ~/.bash_profile
+
+# Download VritualBox repository
+sudo wget http://download.virtualbox.org/virtualbox/rpm/fedora/virtualbox.repo -O /etc/yum.repos.d/virtualbox.repo
+
+# Clean repository cache & import GPG Key
+sudo dnf clean dbcache
+sudo dnf -y repolist        # Import the GPG keys
+
+# Get vagrant version
+export virtualbox_version=`yum -y search VirtualBox | awk 'match($0, /(VirtualBox-[0-9]\.[0-9])/, a) {b=a[1]} END {print b}'`  # depends on yum sorting
+export virtualbox_version=`yum -y search VirtualBox | awk 'match($0, /(VirtualBox-[0-9]\.[0-9])/, a)  {if (RSTART)  { ++i; v[i]=a[1]}} END {n = asort (v); print v[n]}'`
```

Comment:
@smartbit 
why two exports with the same variable name `virtualbox_version` ?
the last one will override the first.